examples/machine-learning/README.md

```diff
@@ -0,0 +1,319 @@
+# Objective
```

Comment:
One thought now, I wonder if we should add A3 somewhere here
today, we have: examples/machine-learning/ml-slurm-h100-2-cluster.yaml
I am wondering how would have to change for different machine types.